Lib/idlelib/help.py

```diff
@@ -209,15 +210,15 @@ class HelpFrame(Frame):
     "Display html text, scrollbar, and toc."
     def __init__(self, parent, filename):
         Frame.__init__(self, parent)
-        # keep references to widgets for test access.
+        # Keep references to widgets for test access.
```

Comment:
I think this comment should be removed. Keeping references like this is done in most of IDLE's classes, and it serves for more than just enabling testing.

Reply:
Done.  It was written years ago and become more routine since.